js/base/Exchange.js

```diff
@@ -1999,4 +1999,19 @@ module.exports = class Exchange {
             throw new NotSupported (this.id + ' ' + key + ' does not have a value in mapping')
         }
     }
+
+    timestampWithinXMonths (timestamp, months) {
```

Comment:
i don't think we should add these base helper methods, this should be done in the derived class imo

Reply:
I can't think of a way to do this without datetime objects, and those aren't transpiled

Reply:
what about 31 * months

most exchanges are not keeping track of the exact number of months i think

Reply:
I'll have to test this out in December and see if anything changes, because it looks like exactly 3 months right now, we'll still need the `dayStart` method though

Reply:
So it turns out that there isn't a 3 month limit, but a limit of 93 days, and if the start date pre-dates today - 93 days, then `endDate` is a required parameter, took me a bit to figure out